.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "solidity/lib/prover-contracts"]
```

Comment:
Please install this dependencies as node.js packages using yarn, we are not using git submodule
https://x.com/PaulRBerg/status/1736695487057531328

Reply:
makes sense! I've replaced the dependency from submodule to npm package.